resources/styles/components/task-step/reading/book-content/index.less

```diff
@@ -25,20 +28,43 @@
       color: @tutor-tertiary;
       background: @tutor-secondary;
       padding: 0 10px;
-      position: absolute;
-      top: 0;
-      left: 0;
       height: @reading-ui-banner-height;
+
     }
-    .title {
-      margin: @reading-ui-banner-height 20px 5px 20px;
-      #fonts > .sans(3rem, 3rem);
-      font-weight: 900;
-      letter-spacing: 2px;
+
+    > [data-type=exercise] {
+
+      content: "";
+      display: block;
+      margin: 0;
+      width: 100%;
+      margin-top: -3px;
+      border-top: 1px solid @tutor-neutral-light;
+
+      > [data-type=title]:first-child {
+        
+        margin: @tutor-card-body-padding/2 @tutor-card-body-padding;
+        #fonts > .sans(3rem, 3rem);
+        font-weight: 900;
+        letter-spacing: 2px;
+      }
+      
+    }
+    .problem, .solution, .commentary {
```

Comment:
That entire block is gone now.  The new mockups aren't using 0 margin on the left/right sides so there's now no need to have the margins set to 0 and then add it back on all the elements.